### PR TITLE
review message

### PR DESCRIPTION
**Pull request by jburel**
Review ``dry run `` message
see #5

### CODE REVIEW

**Comment by pwalczysko** [comment]
The message change in this PR is okay.
But I think you need to rebase on master to fix the failing test - @sbesson pushed some commits today to master, I have tried to rebase your branch on master and it fixed the tests, see https://github.com/pwalczysko/omero-cli-duplicate/actions/runs/8078799489

**Review by pwalczysko — CHANGES_REQUESTED** [review]
Fix the build (probably by rebasing on master)

**Comment by jburel** [comment]
@pwalczysko build is now green

File: src/omero/plugins/duplicate.py [hunk 1]
```diff
@@ -45,11 +45,10 @@
     # Do the same reporting all the new duplicate objects
     omero duplicate Dataset:50 --report
 
-    # Do a dry run of a duplicate reporting the outcome
-    # if the duplicate had been run
+    # Print the outputs of the duplicate command, but do not execute it
     omero duplicate Dataset:53 --dry-run
-    # Do a dry run of a duplicate, reporting all the objects
-    # that would have been duplicated
+    # Report all the objects that  would have been duplicated by
```
Comment (pwalczysko) [inline src/omero/plugins/duplicate.py:50]:
```suggestion
    # Report all the objects that would have been duplicated by
```

**Review by pwalczysko — CHANGES_REQUESTED** [review]
one redundant whitespace found https://github.com/ome/omero-cli-duplicate/pull/25/files#r1522965832

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**